pvr.iptvsimple/resources/language/resource.language.en_au/strings.po

```diff
@@ -87,3 +87,31 @@ msgstr "Channels Logos Base URL"
 msgctxt "#30042"
 msgid "Ignore"
 msgstr "Ignore"
+
```

Comment:
You should only update `resource.language.en_gb/strings.po`, the rest are handled by the translation system. So drop all other language changes except for that file.